Basic-Video-Chat/Basic-Video-Chat.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,7 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		406CFBDFBD756E5FFA361051 /* Pods_Basic_Video_Chat.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = C25B9664605A2644AF1BCF12 /* Pods_Basic_Video_Chat.framework */; };
```

Comment:
Pods framework configuration on .pbxproj is not necessary to be included on the PR. Will be overwritten by `pod install`.